taxonomy-editor-frontend/src/pages/allentries/index.jsx

```diff
@@ -29,6 +33,14 @@ const Entry = () => {
     const [openAddDialog, setOpenAddDialog] = useState(false);
     const [openSuccessSnackbar, setOpenSuccessSnackbar] = useState(false);
 
+    // Set url prefix for navbar component
+    useEffect(() => {
+        setDisplayedPages([

```

Comment:
thanks for the comment, really useful with `useEffect`. An alternative is using something like https://sergiodxa.com/articles/pro-tip-name-your-useeffect-functions  , it is not very common but I like it

Reply:
Cool, I added a name for the useEffect